docs/machine-learning/how-to-guides/inspect-intermediate-data-ml-net.md

```diff
@@ -1,75 +1,180 @@
 ---
-title: Inspect intermediate data values during ML.NET pipeline processing
+title: Inspect intermediate data values during ML.NET processing
 description: Learn how to inspect actual intermediate data values during ML.NET machine learning pipeline processing
-ms.date: 03/05/2019
-ms.custom: mvc,how-to
+ms.date: 04/29/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc
+ms.topic: how-to
 #Customer intent: As a developer, I want to inspect actual intermediate data values during ML.NET machine learning pipeline processing so that I can make sure that I'm getting the results I expect.
 ---
 
-# Inspect intermediate data values during ML.NET pipeline processing
+# Inspect intermediate data values during processing
 
-> [!NOTE]
-> This topic refers to ML.NET, which is currently in Preview, and material may be subject to change. For more information, visit [the ML.NET introduction](https://www.microsoft.com/net/learn/apps/machine-learning-and-ai/ml-dotnet).
+Learn how to inspect values during loading, processing and training steps in ML.NET.
 
-This how-to and related sample are currently using **ML.NET version 0.10**. For more information, see the release notes at the [dotnet/machinelearning GitHub repo](https://github.com/dotnet/machinelearning/tree/master/docs/release-notes).
+Data like the one represented below which is loaded into an [`IDataView`](xref:Microsoft.ML.IDataView) can be inspected in various ways in ML.NET.
+ 
+```csharp
+HousingData[] housingData = new HousingData[]
+{
+    new HousingData
+    {
+        Size = 600f,
```

Comment:
>Size [](start = 8, length = 4)

'Size' is a bit confusing b/c like Count/Length, it's usually used for a container size. Here it's just a field, and it might confuse people that a row always has a Size property, which is not true. I prefer to call it something more specific like 'SizeSqft' or 'SquareFootage' or 'Area', to prevent that potential confusion.

Reply:
Good suggestion. Adding to backlog.